entry.go

```diff
@@ -110,6 +110,10 @@ func (entry *Entry) Debug(args ...interface{}) {
 	}
 }
 
+func (entry *Entry) debug(args ...interface{}) {
+	entry.log(DebugLevel, fmt.Sprint(args...))
+}
```

Comment:
Yeah, I don't think this extra code is necessary but I'm happy to accept the other part of the patch. Please rebase when you do that as well so it's just one commit. Thanks!